options/options.go

```diff
@@ -177,6 +197,17 @@ func (o *Options) Validate() error {
 		)
 	}
 
+	// Validate the format markdown is used iff. the --dependencydiff option is not empty.
+	if o.Dependencydiff == "" && o.Format == FormatMarkdown {
+		errs = append(
+			errs,
+			fmt.Errorf(
+				"%w: markdown can be only used for the --dependencydiff option",
+				errFormatNotSupported,
+			),
+		)
+	}
+
```

Comment:
More such validation will be needed if we support this feature as a flag (like`--depdiff` and `--local` can't be used together). Instead a sub-command is cleaner.